### PR TITLE
Register & Login page, Refactor landing page

### PR DESCRIPTION
**Pull request by AdamMindGate**
-refactor: landing page
-feature: Register & Login page

### CODE REVIEW

File: views/landing.html [hunk 1]
```diff
@@ -20,17 +20,24 @@
...
-      </h4>
+      <div class="text">
+        <h1>SHORTLY</h1>
+        <h4>
+          No-Login link shortener website, Simple clicks to make a shortened
+          link.
+        </h4>
+      </div>
+      <div class="image">
+        <img src="test.png" alt="" />
```
Comment (AbdoRayan404) [inline views/landing.html:39]: Can you provide the link for test.png?

Comment (AbdoRayan404) [reply views/landing.html:39]: Or if it was a test picture, delete the img tag

File: views/reg.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          class="input-field"
+          pattern=".{8,}"
+          title="Please write 8 characters password"
+          placeholder="Rewrite your password"
+          required
+        />
+        <input type="checkbox" required />Accept the terms.
+        <button type="submit" class="submit-button">Sign-Up</button>
+        <button type="submit" class="submit-button">
+          Alread have an account? Sign-In
```
Comment (AbdoRayan404) [inline views/reg.html:58]: typo error

File: views/landing.html (hunk 1 above)
Comment (AdamMindGate) [reply views/landing.html]: It was a test picture. I deleted it.

File: views/reg.html (hunk 2 above)
Comment (AdamMindGate) [reply views/reg.html]: Fixed

**Pull request merged**